schema/types/suricata.schema

```diff
@@ -97,17 +97,39 @@ type suricata.dcerpc = suricata.component.common + record {
   }
 }
 
+// https://github.com/OISF/suricata/blob/master/rust/src/dhcp/logger.rs
```

Comment:
Please write this comment in a user-consumable way, since the schema files are user-facing.